src/frontend/js/utils/AbilitiesHelper/joanieUserProfileAbilities.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { JoanieUserApiAbilityActions } from 'types/User';
+import { JoanieUserProfileFactory } from 'utils/test/factories/joanie';
+import { JoanieUserProfileActions } from './types';
+import AbilitiesHelper from '.';
+
+describe('Abilities.JoanieUserProfile', () => {
+  it.each([
+    {
+      label: 'User has access to both a course and an organization',
+      abilities: {
+        [JoanieUserApiAbilityActions.HAS_ORGANIZATION_ACCESS]: true,
+        [JoanieUserApiAbilityActions.HAS_COURSE_ACCESS]: true,
+      },
+    },
+    {
+      label: 'User has access to a course',
+      abilities: {
+        [JoanieUserApiAbilityActions.HAS_ORGANIZATION_ACCESS]: false,
+        [JoanieUserApiAbilityActions.HAS_COURSE_ACCESS]: true,
+      },
+    },
+    {
+      label: 'User has access to an organization',
+      abilities: {
+        [JoanieUserApiAbilityActions.HAS_ORGANIZATION_ACCESS]: true,
+        [JoanieUserApiAbilityActions.HAS_COURSE_ACCESS]: false,
+      },
+    },
+  ])('should allow ACCESS_TEACHER_DASHBOARD when "$label"', ({ abilities }) => {
+    const joanieUser = JoanieUserProfileFactory({
+      abilities,
+    }).one();
+    expect(
+      AbilitiesHelper.can(joanieUser, JoanieUserProfileActions.ACCESS_TEACHER_DASHBOARD),
+    ).toBeTruthy();
+    expect(
+      AbilitiesHelper.cannot(joanieUser, JoanieUserProfileActions.ACCESS_TEACHER_DASHBOARD),
+    ).toBeFalsy();
```

Comment:
`toBeTruthy` or `toBeFalsy` can lead to false positive.
 We tried to remove the use of those methods in our test in favor of `toBe(true)` and `toBe(false)`